TestAssets/TestProjects/MSBuildTestApp/project.json

```diff
@@ -0,0 +1,31 @@
+{
+  "version": "1.0.0-*",
+  "buildOptions": {
+    "debugType": "portable",
+    "emitEntryPoint": true
+  },
+  "dependencies": {},
+  "frameworks": {
+    "netcoreapp1.0": {
```

Comment:
This should be under dependencies, not frameworks - there's no concept of cross compile yet.